<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QMainWindow" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SCHNApps</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/icons/cgogn/cgogn2.png</normaloff>:/icons/cgogn/cgogn2.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionShowHideDock"/>
    <addaction name="actionShowHidePythonDock"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAboutSCHNApps"/>
    <addaction name="actionAboutCGoGN"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionManageCameras"/>
   <addaction name="actionManagePlugins"/>
   <addaction name="actionManageMaps"/>
  </widget>
  <action name="actionAboutCGoGN">
   <property name="text">
    <string>About CGoGN</string>
   </property>
  </action>
  <action name="actionAboutSCHNApps">
   <property name="text">
    <string>About SCHNApps</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionManagePlugins">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/icons/plugins.png</normaloff>:/icons/icons/plugins.png</iconset>
   </property>
   <property name="text">
    <string>Manage plugins</string>
   </property>
  </action>
  <action name="actionManageCameras">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/icons/cameras.png</normaloff>:/icons/icons/cameras.png</iconset>
   </property>
   <property name="text">
    <string>Manage cameras</string>
   </property>
  </action>
  <action name="actionShowHideDock">
   <property name="text">
    <string>Show/Hide Plugins Dock</string>
   </property>
  </action>
  <action name="actionManageMaps">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/icons/maps.png</normaloff>:/icons/icons/maps.png</iconset>
   </property>
   <property name="text">
    <string>ManageMaps</string>
   </property>
   <property name="toolTip">
    <string>Manage maps</string>
   </property>
  </action>
  <action name="actionShowHidePythonDock">
   <property name="text">
    <string>Show/Hide Python Dock</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>Window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
